.github/workflows/avm.template.module.yml

```diff
@@ -133,6 +133,7 @@ jobs:
           customLocation: "${{ fromJson(inputs.workflowInput).customLocation }}"
         env:
           AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+          CI_KEY_VAULT_NAME: ${{ secrets.CI_KEY_VAULT_NAME }}
```

Comment:
Should be undone. If you have tenant-specific secrets you have to set the key vault name up as a repository variable, not a variable secret 😉 
Then, you also don't have to pass the value down explicitely

Reply:
removed line